main.c

```diff
@@ -36,8 +55,13 @@ int main(void)
 
     msg_init_queue(_main_msg_queue, MAIN_QUEUE_SIZE);
 
+    saul_cord_ep_register_cb(_on_ep_event);
```

Comment:
Since we want to make our project includeable as a module, I would keep it like this, because our main module would not be included in an application using our module.